### DISCUSSION TITLE
How do I set TTL for a file, so it is deleted automatically after some time?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [minio/minio#16809] [Q&A] [answered]
How do I set TTL for a file, so it is deleted automatically after some time?
Is there a way to disable TTL if this file is needed in some time in the future?

**Comment by vadmeste** [comment, marked as answer]
You can use ILM expiry feature - https://min.io/docs/minio/linux/administration/object-management/object-lifecycle-management.html#object-expiration
